docker-compose.yml

```diff
@@ -1,6 +1,13 @@
 version: '2'
 
 services:
+  webapp:
```

Comment:
It might be good to add a `depends_on` mongo so we can turn all containers up together.